### ISSUE TITLE
The missing mandatory screenshots issue

### ISSUE THREAD

**Issue by patkap** [patkap/GeoNature-citizen#5]
a really dark theme:
![Capture d’écran du 2019-07-26 11-41-21](https://user-images.githubusercontent.com/11497003/61943639-2d394e00-af9c-11e9-94e1-de71753998e1.png)

**Comment by patkap** [comment]
Là c'est le ponpon !
![Capture d’écran de 2019-06-20 13-06-09 cropped](https://user-images.githubusercontent.com/11497003/61944248-8bb2fc00-af9d-11e9-98df-6060fecf7938.png)

**Comment by patkap** [comment]
just added a heatmap layer you can toggle:
![Capture d’écran du 2019-07-27 13-56-06](https://user-images.githubusercontent.com/11497003/61994108-9db49d80-b076-11e9-868c-9be7f2dee081.png)

**Comment by patkap** [comment]
home: program layout, masonry style ...

![localhost_4200_home](https://user-images.githubusercontent.com/11497003/62924844-3f472900-bdb1-11e9-8513-4a303e7c2a12.png)

**Comment by patkap** [comment]
![localhost_4200_home_masonry4](https://user-images.githubusercontent.com/11497003/62999405-9ced7b00-be6e-11e9-859f-1cb26087c849.jpg)

**Comment by patkap** [comment]
![Screencast from 08-15-2019 04:25:20 PM-optimized](https://user-images.githubusercontent.com/11497003/63102553-a9132e80-bf7b-11e9-92b2-17cf3abbab2f.gif)

**Comment by patkap** [comment]
![Screencast from 08-17-2019T05:36:52-optimize](https://user-images.githubusercontent.com/11497003/63206385-7ff5b980-c0b3-11e9-95b1-6915afd88704.gif)

**Comment by lpofredc** [comment]
Salut @patkap , 
Certaines fonctions sont très intéressante, notamment celle alertant lorsque la donnée est hors zone. Une PR est-elle envisagée?

Merci.

> ![Screencast from 08-15-2019 04:25:20 PM-optimized](https://user-images.githubusercontent.com/11497003/63102553-a9132e80-bf7b-11e9-92b2-17cf3abbab2f.gif)

**Comment by samuelpriou** [comment]
> 
> 
> Salut @patkap ,
> Certaines fonctions sont très intéressante, notamment celle alertant lorsque la donnée est hors zone. Une PR est-elle envisagée?
> 
> Merci.
> 
> > ![Screencast from 08-15-2019 04:25:20 PM-optimized](https://user-images.githubusercontent.com/11497003/63102553-a9132e80-bf7b-11e9-92b2-17cf3abbab2f.gif)

C'est effectivement une fonctionnalité intéressante !

**Comment by patkap** [comment]
> Salut @patkap ,
> Certaines fonctions sont très intéressante, notamment celle alertant lorsque la donnée est hors zone. Une PR est-elle envisagée?
> 
> Merci.
> 
Oui, bien sûr, dès que j'aurai revu les opérations pour les effectuer dans le bon sens et consolidé la fonctionnalité!

**Comment by samuelpriou** [comment]
Super ! Merci Patrick.

**Comment by patkap** [comment]
15k d'obs, s'il vous plaît, mon bon Monsieur!
https://citizendemo.patkap.tech/programs/26/observations#observations
![image](https://user-images.githubusercontent.com/11497003/69666144-10b53b80-108c-11ea-81af-4d8db340182b.png)
